project-templates/windows/1/rancher-compose.yml

```diff
@@ -0,0 +1,4 @@
+.catalog:
+  name: Windows
+  version: 0.0.2
+  minimum_rancher_version: "v1.6.11"
```

Comment:
`v1.6.13-rc1`

Reply:
In the rancher-compose.yml in the `0` folder for windows, can you add in `maximum_rancher_version: v1.6.12`?